superduperdb/container/model.py

```diff
@@ -400,6 +405,7 @@ class Model(Component, PredictMixin):
     version: t.Optional[int] = None
     future: t.Optional[Future] = None
     device: str = "cpu"
+    preferred_devices: t.List[str] = ["cpu", "cuda", "mps"]
```

Comment:
I would reverse the order and break the `for` loop if one is successful already.

Reply:
Addressed in latest commit :)